include/SFML/Window/WindowBase.hpp

```diff
@@ -157,7 +157,7 @@ class SFML_WINDOW_API WindowBase
     bool isOpen() const;
 
     ////////////////////////////////////////////////////////////
-    /// \brief Pop the event on top of the event queue, if any, and return it
+    /// \brief Pop the event from top of the event queue, if any, and return it
```

Comment:
```suggestion
    /// \brief Pop the event from the event queue, if any, and return it
```

Reply:
Nicer, indeed! (and more correct, since queue is no stack, where "top" is the actual one location of adding/removing)

Reply:
I don't necessarily agree. Yes, it's kind of implicit that you take it one at a time from the queue, but being specific with "top" indicates a clear order. We're not taking any events from the bottom, but also not in end middle, but one event from the top after the other.

Reply:
Another thing -- queues usually use "front" and "back" terminology, while "top" and "bottom" is used for stacks.

Might also be worth mentioning that it's a FIFO queue, if we're already going that far.

Reply:
@tverrbjelke can you change it to:

> Pop the next event from the front of the FIFO event queue, if any, and return it